pontoon/base/tests/test_models.py

```diff
@@ -98,11 +98,20 @@ def setUp(self):
         EntityFactory.create(resource=self.resource)
         StatsFactory.create(resource=self.resource, locale=self.locale)
 
+    def _call_test_method(self):
```

Comment:
`fetch_locales_parts_stats` is a better name for this, it explains what it's actually doing.